cli/examples/package-authoring/dynatrace-operator/README.md

```diff
@@ -0,0 +1,25 @@
+# dynatrace-operator
+
+This example demonstrate how to package [`dynatrace-operator`](https://github.com/Dynatrace/dynatrace-operator) Kubernetes manifest. Dynatrace operator releases `kubernetes.yaml` file which can be packaged into a Carvel package. `kubernetes.yml` can be fetched from the [`github release artifact`](https://github.com/Dynatrace/dynatrace-operator/releases). 
+
+This example uses the `v0.6.0` release.
+
+## Change Release version
+
+In case you want to use some other release version, rerun `pkg init`. During rerun, `kctrl` will read the values from `package-build.yml`, `vendir.yml` and present them as default values. When asked for the `Release tag`, enter the desired release version.
```

Comment:
Should we be describing how the resources were put together instead of an update flow?

Reply:
This is already part of package authoring commands tutorial docs.